### PR TITLE
Standardize XEH and Versioning logging prefixs

### PR DESCRIPTION
**Pull request by PabstMirror**
before
```
XEH: missionNamespace processed [false]
[CBA] (settings) INFO: Checking mission settings file ...
[3876,80.609,0,"XEH: PreInit started. v3.13.0.191116"]
[CBA] (settings) INFO: Reading settings from settings file.
[3878,81.513,0,"XEH: PostInit started. MISSIONINIT: missionName=tempMissionSP, missionVersion=53, worldName=Stratis, isMultiplayer=false, isServer=true, isDedicated=false, CBA_isHeadlessClient=false, hasInterface=true, didJIP=false"]
[3878,81.528,0,"CBA_VERSIONING: cba=3.13.0.191116, ace=3.12.6.43, acex=3.4.2.13, "]
[3878,81.919,0,"XEH: PostInit finished."]
[CBA] (optics) INFO: Scripted camera restarted.
```
after
```
[CBA] (XEH): missionNamespace processed [false]
[CBA] (XEH) [1998,49.636,0] PreInit started. v3.13.0.191116
[CBA] (settings) INFO: Reading settings from settings file.
[CBA] (settings) INFO: Finished reading settings from settings file.
[CBA] (XEH) [1998,50.153,0] PreInit finished.
[CBA] (XEH) [2000,50.587,0] PostInit started. MISSIONINIT: missionName=tempMissionSP, missionVersion=53, worldName=Stratis, isMultiplayer=false, isServer=true, isDedicated=false, CBA_isHeadlessClient=false, hasInterface=true, didJIP=false
[CBA] (versioning) [2000,50.602,0] cba=3.13.0.191116, ace=3.12.6.43, acex=3.4.2.13
[CBA] (XEH) [2000,50.996,0] PostInit finished.
[CBA] (optics) INFO: Scripted camera restarted.
```
Makes logging consistent with other CBA
Some people may not know that XEH means CBA 
(although most who know how to open a RPT probably do)

### CODE REVIEW

**Comment by commy2** [comment]
Why not https://github.com/CBATeam/CBA_A3/blob/master/addons/main/script_macros_common.hpp#L239-L247 if you want consistency?

**Comment by PabstMirror** [comment]
I think that needs DEBUG_SYNCHRONOUS

**Comment by commy2** [comment]
Then define `DEBUG_SYNCHRONOUS` (should be already for xeh component).
There is no reason not to.

File: addons/xeh/fnc_addClassEventHandler.sqf [hunk 1]
```diff
@@ -33,7 +33,7 @@ private _config = configFile >> "CfgVehicles" >> _className;
 
 // init fallback loop when executing on incompatible class for the first time
 if (!GVAR(fallbackRunning) && {ISINCOMP(_className)}) then {
-    diag_log text format ["[XEH]: One or more children of class %1 do not support Extended Event Handlers. Fall back to loop.", configName _config];
+    WARNING_1("One or more children of class %1 do not support Extended Event Handlers. Fall back to loop.", configName _config);
```
Comment (PabstMirror) [inline addons/xeh/fnc_addClassEventHandler.sqf]: `[CBA] (xeh) WARNING: One or more children of class All do not support Extended Event Handlers. Fall back to loop. x\cba\addons\xeh\fnc_addClassEventHandler.sqf:36`

is WARNING ok here? it does add the fileLineNumber stuff at the end

Comment (commy2) [reply addons/xeh/fnc_addClassEventHandler.sqf]: WARNING requires DEBUG_MODE_NORMAL.

Comment (PabstMirror) [reply addons/xeh/fnc_addClassEventHandler.sqf]: `[CBA] (xeh) WARNING: One or more children of class "All" do not support Extended Event Handlers. Fall back to loop.`

**Pull request merged**